app_guides/php/overview.adoc

```diff
@@ -0,0 +1,37 @@
+[[appguides-php-overview]]
+= PHP Application Hosting
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+[.lead]
+PHP is a popular general-purpose scripting language that is especially suited to
+web development.
+{product-title} currently supports PHP 5.5 and 5.6.
+
+image::php-logo.png[PHP logo]
+
+[NOTE]
+====
+Learn more at the official link:http://www.php.net[PHP site].
```

Comment:
I would take this NOTE box out and make the first instance of PHP on the page a link